### PR TITLE
Added new pack: Email Hippo

### PR DESCRIPTION
**Pull request by ilappe**
## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-6673

## Description
a new pack `Email Hippo` including:
- Email Hippo integration

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/EmailHippo/Integrations/EmailHippo/README.md [hunk 1]
```diff
@@ -0,0 +1,505 @@
+This is the Email Hippo integration for verify email sources as fake emails that were used as part of phishing attacks.
```
Comment (ShirleyDenkberg) [inline Packs/EmailHippo/Integrations/EmailHippo/README.md:1]:
```suggestion
This is the Email Hippo integration used to verify email sources as fake emails that were used as part of phishing attacks.
```

File: Packs/EmailHippo/Integrations/EmailHippo/README.md [hunk 2]
```diff
@@ -0,0 +1,505 @@
...
+## Configure Email Hippo on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for Email Hippo.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | MORE Server URL (e.g. https://api.hippoapi.com) |  | True |
+    | Email Hippo WHOIS Server URL (e.g. https://api.whoishippo.com) |  | True |
```
Comment (ShirleyDenkberg) [inline Packs/EmailHippo/Integrations/EmailHippo/README.md:14]:
```suggestion
    | MORE Server URL (e.g., https://api.hippoapi.com) |  | True |
    | Email Hippo WHOIS Server URL (e.g., https://api.whoishippo.com) |  | True |
```

File: Packs/EmailHippo/Integrations/EmailHippo/README.md [hunk 3]
```diff
@@ -0,0 +1,505 @@
...
+
+## Commands
+
+You can execute these commands from the Cortex XSOAR CLI, as part of an automation, or in a playbook.
+After you successfully execute a command, a DBot message appears in the War Room with the command details.
+
+### email-hippo-email-quota-get
+
+***
+Get the Email quota from the API.
```
Comment (ShirleyDenkberg) [inline Packs/EmailHippo/Integrations/EmailHippo/README.md:32]:
```suggestion
Get the email quota from the API.
```

File: Packs/EmailHippo/Integrations/EmailHippo/README.md [hunk 4]
```diff
@@ -0,0 +1,505 @@
...
+>### Email quota
+>|Email Quota remaining|Email Quota used|
+>|---|---|
+>| 99 | 1 |
+
+
+### email
+
+***
+Return Email information and reputation.
```
Comment (ShirleyDenkberg) [inline Packs/EmailHippo/Integrations/EmailHippo/README.md:78]:
```suggestion
Return email information and reputation.
```

File: Packs/EmailHippo/Integrations/EmailHippo/README.md [hunk 5]
```diff
@@ -0,0 +1,505 @@
...
+
+#### Base Command
+
+`email`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| email | The email address to validate (CSV). | Required |
```
Comment (ShirleyDenkberg) [inline Packs/EmailHippo/Integrations/EmailHippo/README.md:88]:
```suggestion
| email | A comma-separated list of email addresses to validate. | Required | 
```

File: Packs/EmailHippo/Integrations/EmailHippo/README.md [hunk 6]
```diff
@@ -0,0 +1,505 @@
...
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | MORE Server URL (e.g. https://api.hippoapi.com) |  | True |
+    | Email Hippo WHOIS Server URL (e.g. https://api.whoishippo.com) |  | True |
+    | MORE API Key |  | True |
+    | WHOIS API Key |  | True |
+    | Source Reliability | Reliability of the source providing the intelligence data. | False |
+    | Create relationships | Create relationships between indicators as part of Enrichment. | False |
```
Comment (ShirleyDenkberg) [inline Packs/EmailHippo/Integrations/EmailHippo/README.md:18]:
```suggestion
    | Create relationships | Create relationships between indicators as part of enrichment. | False |
```

File: Packs/EmailHippo/Integrations/EmailHippo/README.md [hunk 7]
```diff
@@ -0,0 +1,505 @@
...
+>### Email test@example.com
+>|Hippo Trust Score|Inbox quality score|Result|Spam risk score|
+>|---|---|---|---|
+>| Low | DoNotSend | result: Unverifiable<br/>reason: DomainIsWellKnownDea | Block |
+
+
+### domain
+
+***
+Returns Domain information and reputation.
```
Comment (ShirleyDenkberg) [inline Packs/EmailHippo/Integrations/EmailHippo/README.md:253]:
```suggestion
Returns domain information and reputation.
```

File: Packs/EmailHippo/Integrations/EmailHippo/README.md [hunk 8]
```diff
@@ -0,0 +1,505 @@
...
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| DBotScore.Indicator | String | The indicator that was tested. | 
+| DBotScore.Reliability | String | The reliability score of the vendor. | 
+| DBotScore.Score | Number | The actual score. | 
+| DBotScore.Type | String | The indicator type. | 
+| DBotScore.Vendor | String | The vendor used to calculate the score. | 
+| Domain.Name | String | The name of the domain that was checked. | 
+| Domain.NameServers | String | Name servers of the domain. |
```
Comment (ShirleyDenkberg) [inline Packs/EmailHippo/Integrations/EmailHippo/README.md:275]:
```suggestion
| Domain.NameServers | String | Name of the servers of the domain. | 
```

File: Packs/EmailHippo/Integrations/EmailHippo/README.md [hunk 9]
```diff
@@ -0,0 +1,505 @@
...
+| --- | --- | --- |
+| DBotScore.Indicator | String | The indicator that was tested. | 
+| DBotScore.Reliability | String | The reliability score of the vendor. | 
+| DBotScore.Score | Number | The actual score. | 
+| DBotScore.Type | String | The indicator type. | 
+| DBotScore.Vendor | String | The vendor used to calculate the score. | 
+| Domain.Name | String | The name of the domain that was checked. | 
+| Domain.NameServers | String | Name servers of the domain. | 
+| Domain.UpdatedDate | Date | The date that the domain was last updated. | 
+| Domain.CreationDate | Date | The creation date of the domain. Format is ISO8601 \(i.e. '2020-04-30T10:35:00.000Z'\). |
```
Comment (ShirleyDenkberg) [inline Packs/EmailHippo/Integrations/EmailHippo/README.md:277]:
```suggestion
| Domain.CreationDate | Date | The creation date of the domain. Format is ISO8601 \(i.e.,'2020-04-30T10:35:00.000Z'\). | 
```

File: Packs/EmailHippo/Integrations/EmailHippo/README.md [hunk 10]
```diff
@@ -0,0 +1,505 @@
...
+| Domain.Registrar.Name | String | The name of the registrar. | 
+| Domain.Registrar.AbuseEmail | String | The email address of the contact for reporting abuse. | 
+| Domain.Registrar.AbusePhone | String | The phone number of contact for reporting abuse. | 
+| Domain.Admin.Country | String | The country of the domain administrator. | 
+| Domain.Admin.Email | String | The email address of the domain administrator. | 
+| Domain.Admin.Name | String | The name of the domain administrator. | 
+| Domain.Admin.Phone | String | The phone number of the domain administrator. | 
+| Domain.Tech.Country | String | The country of tech administrator. | 
+| Domain.Tech.Name | String | The name of the tech administrator. | 
+| Domain.Tech.Email | String | The email of tech administrator. |
```
Comment (ShirleyDenkberg) [inline Packs/EmailHippo/Integrations/EmailHippo/README.md:287]:
```suggestion
| Domain.Tech.Email | String | The email of the tech administrator. | 
```

File: Packs/EmailHippo/Integrations/EmailHippo/README.md [hunk 11]
```diff
@@ -0,0 +1,505 @@
...
+| DBotScore.Score | Number | The actual score. | 
+| DBotScore.Type | String | The indicator type. | 
+| DBotScore.Vendor | String | The vendor used to calculate the score. | 
+| Domain.Name | String | The name of the domain that was checked. | 
+| Domain.NameServers | String | Name servers of the domain. | 
+| Domain.UpdatedDate | Date | The date that the domain was last updated. | 
+| Domain.CreationDate | Date | The creation date of the domain. Format is ISO8601 \(i.e. '2020-04-30T10:35:00.000Z'\). | 
+| Domain.Registrar.Name | String | The name of the registrar. | 
+| Domain.Registrar.AbuseEmail | String | The email address of the contact for reporting abuse. | 
+| Domain.Registrar.AbusePhone | String | The phone number of contact for reporting abuse. |
```
Comment (ShirleyDenkberg) [inline Packs/EmailHippo/Integrations/EmailHippo/README.md:280]:
```suggestion
| Domain.Registrar.AbusePhone | String | The phone number of the contact for reporting abuse. | 
```

File: Packs/EmailHippo/Integrations/EmailHippo/README.md (hunk 10 above)
Comment (ShirleyDenkberg) [inline Packs/EmailHippo/Integrations/EmailHippo/README.md:287]:
```suggestion
| Domain.Tech.Email | String | The email of the tech administrator. | 
```

File: Packs/EmailHippo/Integrations/EmailHippo/README.md [hunk 12]
```diff
@@ -0,0 +1,505 @@
...
+| Domain.Admin.Email | String | The email address of the domain administrator. | 
+| Domain.Admin.Name | String | The name of the domain administrator. | 
+| Domain.Admin.Phone | String | The phone number of the domain administrator. | 
+| Domain.Tech.Country | String | The country of tech administrator. | 
+| Domain.Tech.Name | String | The name of the tech administrator. | 
+| Domain.Tech.Email | String | The email of tech administrator. | 
+| Domain.Tech.Organization | String | The organization of the tech administrator. | 
+| Domain.WHOIS.NameServers | String | A CSV string of name servers, for example 'ns1.bla.com, ns2.bla.com'. | 
+| Domain.WHOIS.CreationDate | Date | The creation date of the domain. Format is ISO8601 \(i.e. '2020-04-30T10:35:00.000Z'\). | 
+| Domain.WHOIS.UpdatedDate | Date | The date when the domain was last updated. Format is ISO8601 \(i.e. '2020-04-30T10:35:00.000Z'\). |
```
Comment (ShirleyDenkberg) [inline Packs/EmailHippo/Integrations/EmailHippo/README.md:291]:
```suggestion
| Domain.WHOIS.CreationDate | Date | The creation date of the domain. Format is ISO8601 \(i.e., '2020-04-30T10:35:00.000Z'\). | 
| Domain.WHOIS.UpdatedDate | Date | The date when the domain was last updated. Format is ISO8601 \(i.e., '2020-04-30T10:35:00.000Z'\). | 
```

**Comment by ShirleyDenkberg** [comment]
@michalgold @DeanArbel @idovandijk Doc review completed.

**Comment by ilappe** [comment]
> @ilappe Don't forget to create content for the pack readme.

Thanks @ShirleyDenkberg
i create it 
can you please review it ?

File: Packs/EmailHippo/README.md [hunk 13]
```diff
@@ -0,0 +1,6 @@
+Email Hippo is an email intelligence and data services provider that deliver accurate cloud-based email validation and domain profiling.
```
Comment (ShirleyDenkberg) [inline Packs/EmailHippo/README.md:1]:
```suggestion
Email Hippo is an email intelligence and data services provider that delivers accurate cloud-based email validation and domain profiling. 
```

**Comment by ShirleyDenkberg** [comment]
@ilappe I reviewed the pack readme.  One small edit.

**Review by DeanArbel — APPROVED** [review]
Impressive work, well done.
Let's do a demo.

**Pull request merged**